Formula/doctl.rb

```diff
@@ -30,6 +30,11 @@ def install
     ].join(" ")
     system "go", "build", "-ldflags", ldflags, "github.com/digitalocean/doctl/cmd/doctl"
     bin.install "doctl"
+
+    output = Utils.popen_read("#{bin}/doctl completion bash")
+    (bash_completion/"doctl").write output
```

Comment:
Sorry this fell of our radar.  These lines can be combined like this:
```ruby
(bash_completion/"doctl").write `#{bin}/doctl completion bash`
```
And likewise for zsh.